js/controllers/menu.js

```diff
@@ -17,19 +17,33 @@ function MenuController(tabs) {
   $(document).bind('tabsave', this.onTabSave.bind(this));
 }
 
-MenuController.prototype.onNewTab = function(e, tab) {
-  var id = tab.getId();
-  var name = tab.getName();
-  var listItem = $('<li id="tab' + id + '" draggable="true">' +
-                   '<div class="filename">' + name + '</div>' +
-                   '<div class="close"></div></li>');
-  listItem.bind('dragstart', this.onDragStart_.bind(this, listItem));
-  listItem.bind('dragover', this.onDragOver_.bind(this, listItem));
-  listItem.bind('dragend', this.onDragEnd_.bind(this, listItem));
-  listItem.bind('drop', this.onDrop_.bind(this, listItem));
-  listItem.appendTo($('#tabs-list'));
-  listItem.click(this.tabButtonClicked_.bind(this, id));
-  listItem.find('.close').click(this.closeTabClicked_.bind(this, id));
+MenuController.prototype.addNewTab_ = function(e, tab) {
+  let id = tab.getId();
+  let tabElement = document.createElement("li");
+  tabElement.setAttribute('draggable', 'true');
+  tabElement.id = 'tab' + id;
+  let filenameElement = document.createElement("div");
+  filenameElement.appendChild(document.createTextNode(tab.getName()));
+  filenameElement.className = "filename";
+  tabElement.appendChild(filenameElement);
+  let closeElement = document.createElement("div");
+  closeElement.setAttribute('title', chrome.i18n.getMessage('closeFileButton'))
+  closeElement.className = "close";
+  tabElement.appendChild(closeElement);
+  document.getElementById('tabs-list').appendChild(tabElement);
+
+  tabElement.addEventListener(
```

Comment:
Is this file ES6 ready? Can you use arrow functions, rather than `this.onDragStart_.bind(this, $(tabElement)));`

Instead do

`(event) => { this.onDragStart_($(tabElement), event); }`

Reply:
Done.